packages/color-mode/src/index.ts

```diff
@@ -1,3 +1,5 @@
 export * from "./color-mode"
 export * from "./initialize-color-mode"
 export type { ColorModeOptions } from "./use-color-mode-state"
+export { cookieStorageManager, localStorageManager } from "./color-mode.utils"
+export type { StorageManager } from "./color-mode.utils"
```

Comment:
You could put these in `storage-manager.ts` instead so then you can have a barrel export for that file instead. Would make it easier to manage the exports than juggling which pieces from `utils` are exported.